conf.yaml

```diff
@@ -2000,6 +1945,66 @@ contents:
               -
                 repo:   docs
                 path:   shared/attributes.asciidoc
+          - title:      Logs Monitoring Guide for 7.5-7.9
+            prefix:     en/logs/guide
+            current:    7.9
+            branches:   [ 7.9, 7.8, 7.7, 7.6, 7.5 ]
+            live:       *stacklive
```

Comment:
I think we'll want to take `live` out of the configuration of all three books (`stacklive` will eventually diverge to the point where it won't overlap the list of branches in this book). If it's not provided, `live` defaults to the full list of branches, which is fine for something in the Legacy section.

Reply:
Thanks, will do.